packit_service/constants.py

```diff
@@ -262,3 +262,14 @@ def from_number(number: int):
 # shared sandcastle dir
 SANDCASTLE_DG_REPO_DIR = "dist-git"
 SANDCASTLE_LOCAL_PROJECT_DIR = "local-project"
+
+ERROR_COMMENT_MESSAGE_DETAILS = (
+    "commit_sha",
+    "copr_build_packit_info",
+    "srpm_build_packit_info",
+    "copr_build_logs",
+    "koji_build_packit_info",
+    "koji_build_logs",
+    "testing_farm_packit_info",
+    "testing_farm_logs",
```

Comment:
I am wondering if we could consolidate these only to something like `packit_dashboard_url` and `logs_url` :thinking: e.g. when someone configures a Copr build job, it can fail either in the `SRPM` phase or Copr build itself and it would be probably easier for user to configure only something like 

`One of the Copr builds failed for commit {commit_sha}, ping @admin, packit dashboard: {packit_dashboard_url}, logs: {logs_url}`

Reply:
I am on the fence here if we should support both specific and generic ones or just the generic one.

Reply:
> What do you mean for the generic ones 😅?

Sorry, I didn't know how to call the ones Laura suggested -- like `logs_url`  (unlike the "specific" ones like  `copr_build_url`)...

Reply:
Thinking more about it I agree with Laura, the user can't really use in a message `logs_url` coming from different services or a `packit_dashboard_url` for a different job... For this reason I think we don't need to differentiate them and we can make life easier for the user having only two placeholders: `packit_dashboard_url` and `logs_url`.

Reply:
:+1: Makes sense -- and we can easily add more if people ask for that.

Reply:
Maybe, I would provide and differentiate also (not sure about the names) `web_url` and `logs_url`. (We have both for Copr -- on links to Copr frontend view, one to log.)